.github/workflows/chromatic.ds-react-core-form.yml

```
@@ -0,0 +1,11 @@
+name: "react-ds-core/chromatic"
+on:
+  pull_request:
+    paths: [packages/react/ds-core/**]
+jobs:
+  test-compile:
+    uses: ./.github/workflows/chromatic._template.yaml
+    secrets:
+      PROJECT_TOKEN: ${{ secrets.CHROMATIC_TOKEN_CANONICAL_REACT_DS_CORE_FORM }}
+    with:
+      working-directory: packages/react/ds-core
```

Core and form have been mixed up
```suggestion
      working-directory: packages/react/ds-core-form
```